worker/games.py

```diff
@@ -396,7 +396,7 @@ def verify_signature(engine, signature, active_cores):
 
         if active_cores > 1:
             busy_process.communicate("quit\n")
-
+    p.returncode=1
```

Comment:
is this an intended change (the if's after this look strange).

Reply:
Oops. Debug (to generate an `api/stop_run`).